vault/hcp_link/proto/node_status/status.proto

```diff
@@ -7,6 +7,10 @@ package hashicorp.vault.hcp_link.node_status;
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/empty.proto";
 
+message RaftStatus {
+  bool IsVoter = 1;
```

Comment:
maybe adding nodeID as well?

Reply:
My thought is that it would be redundant as it would be the node ID in the node status. I'm not sure that we would end up using it specifically from this message. What are your thoughts?

cc @biazmoreira @divyaac

Reply:
I believe NodeId already exists in NodeStatusResponse, so I don't believe we'll need that.

Reply:
Is there a reason why chose for this to be a message instead of enum (there might have been a discussion about this but I can't quite remember)